Channel category updates (#5250)
mattermost/docs: source/messaging/organizing-your-sidebar.rst

Organizing channels in categories
---------------------------------

> Once you've created categories, you can move channels within them to organize your sidebar. 

Reviewer: ```suggestion
Once you've created categories, you can move channels around to organize your sidebar.
```

Moving selections
~~~~~~~~~~~~~~~~~

> In addition to selecting and dragging, you can specify a category destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.  

Reviewer: ```suggestion
In addition to selecting and dragging, you can specify a category destination for selected channels and/or Direct Messages. To do this, select the **Channel options** icon in the sidebar and then select **Move to**.
```

.. image:: ../images/multi-select-move.gif

> From Mattermost v6.1, you can also specify a category destination for the current channel or conversation using the **Move to** option directly from the Channel Header. Channels moved a category display a checkmark next to the category name.

Reviewer: ```suggestion
From Mattermost v6.1, you can also specify a category destination for the current channel or conversation using the **Move to** option directly from the channel header. Channels that have been moved a category will display a checkmark next to the category name.
```
